rpc/rpc-wrappers.go

```diff
@@ -1201,13 +1201,7 @@ func NewStorageAuthorityServer(rpc Server, impl core.StorageAuthority) error {
 		}
 
 		err = impl.AddSCTReceipt(core.SignedCertificateTimestamp(sct))
-		if err != nil {
-			// AUDIT[ Error Conditions ] 9cc4d537-8534-4970-8665-4b382abe82f3
-			errorCondition(MethodAddSCTReceipt, err, req)
```

Comment:
The method is returning nil in the regular case, but the dbMap.Insert() call can still error, so it's not correct to remove this.